build.sbt

```diff
@@ -382,3 +385,11 @@ lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
       </developer>
     </developers>)
 )
+
+
+def scalaVersionAtLine(n: Int): String = {
```

Comment:
would it be possible to set the scala versions as system properties and read them as we do with the spark flag?

Reply:
I was thinking about that and gave it a try but ended up with another solution